membership-common/src/main/scala/com/gu/memsub/BillingSchedule.scala

```diff
@@ -36,14 +36,13 @@ case class BillingSchedule(invoices: NonEmptyList[Bill]) {
 
 object BillingSchedule {
 
-  type ProductFinder = ProductRatePlanChargeId => Option[Benefit]
   case class BillItem(name: String, product: Option[Benefit], amount: Float, unitPrice: Float) {
     override lazy val toString = s"| $amount\t| ($unitPrice)\t| $name | ${product.mkString}"
   }
 
   object BillItem {
-    def fromItem(finder: ProductFinder)(in: PreviewInvoiceItem) =
-      BillItem(in.chargeName, finder(ProductRatePlanChargeId(in.productRatePlanChargeId)), in.price, in.unitPrice)
+    def fromItem(planMap: Map[ProductRatePlanChargeId, Benefit])(in: PreviewInvoiceItem) =
```

Comment:
although technically more complex, it seemed less confusing to use a built in data type rather than aliasing a function